plugins/cua/vendor/cua-driver/source/Tests/integration/conftest.py

```
@@ -0,0 +1,282 @@
+# ── focus monitor ─────────────────────────────────────────────────────────────
+
+def _build_focus_app() -> None:
+    if not os.path.exists(_FOCUS_APP_EXE):
+        build_sh = os.path.join(_FOCUS_APP_DIR, "build.sh")
+        subprocess.run([build_sh], check=True)
+
+
+def _launch_focus_app():
+    """Launch FocusMonitorApp, return (proc, pid)."""
+    _build_focus_app()
+    proc = subprocess.Popen(
```

Ruff (0.15.12)

[error] `subprocess` call: check for execution of untrusted input

(S603)